src/Compilers/CSharp/Portable/CodeGen/EmitArrayInitializer.cs

```diff
@@ -52,7 +53,7 @@ private void EmitArrayInitializers(ArrayTypeSymbol arrayType, BoundArrayInitiali
             else
             {
                 ImmutableArray<byte> data = this.GetRawData(initExprs);
-                _builder.EmitArrayBlockInitializer(data, inits.Syntax, _diagnostics.DiagnosticBag);
+                _builder.EmitArrayBlockInitializer(data, alignment: 1, inits.Syntax, _diagnostics.DiagnosticBag); // alignment == 1 as there's no special need for alignment (.pack) here.
```

Comment:
>alignment: 1 [](http://example.com/codeflow?start=57&length=12)

Could this default potentially prevent sharing of the blob for a scenario when we could use the "right" specific alignment?

Reply:
This could lead to not sharing a blob across an array and a span created with the same constant data, e.g.
```C#
long[] array = new long[] { 1, 2, 3, 4, 5, 6, 7, 8 };
ReadOnlySpan<long> span = new long[] { 1, 2, 3, 4, 5, 6, 7, 8 };
```
will end up with two distinct blobs. The flip side, however, is that changing this alignment for arrays could lead to a rewriter (e.g. illink) that respects the .pack field regressing size a bit because it'll start enforcing that larger alignment.  I didn't want to mess with that right now, as I don't think we know yet whether that's a good tradeoff.

Reply:
Consider capturing this information in a comment.